private/coursier_utilities.bzl

```diff
@@ -27,7 +27,7 @@ SUPPORTED_PACKAGING_TYPES = [
     "scala-jar",
 ]
 
-def strip_packaging_and_classifier(coord):
+def get_versioned_coord(coord):
```

Comment:
Why these name changes?  Feels like the new names are not as clear.

Reply:
can revert renaming. New names describe the purpose and applicability (maybe even better would be `get_versioned_label_name` and `get_simple_label_name`? Didn't do that initially because the jump to labels requires an extra `escape` on top and there are places where `escape` isn't applied) and it makes it more clear why they can be reused to parse coord strings into labels, while old ones describe implementation (and applicability but for cases where we have purpose in terms of implementation / action on coord parts rather than in terms of ways to address one and the same artifact)

Reply:
reverted the rename. The only place without escape is around checking against jetify_include_list which could be shifted to operate on labels rather than coords, and it may add support for classifier to jetify_include_list, but that can be a separate PR if needed at all

Reply:
thanks, the diff is easier to follow without the renames.